### PR TITLE
chore(docs): improving rust example usability

### PR DESCRIPTION
**Pull request by danielhe4rt**
Resolves: #133 #134 #135 

Fixed:
- Syntax Highlight for Rust, JSON and CQL codeblocks;
- Added information to retrieve Pet/Owner ID's for lookup by API requests;
- Adjusted bash code blocks to copy/paste.

### CODE REVIEW

File: docs/build-with-rust.md [hunk 1]
```diff
@@ -151,7 +152,7 @@ CREATE TABLE carepet.pet (
 
 The sensor service simulates the collar's activity and periodically saves data to the database. Use the below commands to run the sensor service:
 
-```
+```bash
 $ NODE1=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' carepet-scylla1)
```
Comment (zseta) [inline docs/build-with-rust.md:156]:
```suggestion
NODE1=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' carepet-scylla1)
```

Comment (zseta) [inline docs/build-with-rust.md:156]:
```suggestion
NODE1=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' carepet-scylla1)
```

File: docs/build-with-rust.md [hunk 2]
```diff
@@ -151,7 +152,7 @@ CREATE TABLE carepet.pet (
...
 $ cargo run --bin sensor -- --hosts $NODE1 --measure 5s --buffer-interval 1m
```
Comment (zseta) [inline docs/build-with-rust.md:157]:
```suggestion
cargo run --bin sensor -- --hosts $NODE1 --measure 5s --buffer-interval 1m
```

File: docs/build-with-rust.md [hunk 3]
```diff
@@ -351,14 +353,77 @@ The GET request on URL `/sensor/<id>/values?<from>&<to>` triggers `find_sensor_d
...
+```
+
+If you want to list owner's pets you can use the following command:
+
+```shell
+curl http://127.0.0.1:8000/owner/{id}/pets
+```
 
-To test out the API in your terminal, use the following command: `$ curl http://127.0.0.1:8000/api/owner/{id}` and expect the following response:
+and you should receive a response with something like:
```
Comment (zseta) [inline docs/build-with-rust.md:382]:
```suggestion
and you should receive a response similar to this:
```

**Review by zseta — CHANGES_REQUESTED** [review]
Found some $ characters at the beginning of commands

**Comment by DanielHe4rt** [comment]
@zseta thanks! Fixed and ready to go.

**Review by zseta — APPROVED** [review]

**Pull request merged**